projects/storefrontlib/src/lib/ui/layout/header/mobile-menu/mobile-menu.component.ts

```diff
@@ -11,6 +11,7 @@ import { Subscription } from 'rxjs';
 export class MobileMenuComponent implements OnDestroy {
   private subscription: Subscription;
   showMenu = false;
+  canAddToHomeScreen = false;
```

Comment:
```suggestion
```

Right, was left from prev solution. Added a suggestion for @marlasak  (btw this new suggestions feature is neat)